spec/reporters/muted-console-reporter.js

```diff
@@ -0,0 +1,58 @@
+const util = require("util");
+const fs = require("fs");
+
+const originalLog = console.log;
```

Comment:
capturing only `console.log()` and `console.error()` neglects all other console methods. It would be better to replace the console wholesale.

See https://nodejs.org/docs/latest/api/console.html#console_new_console_stdout_stderr_ignoreerrors